Documentation/ceph-nfs-crd.md

```diff
@@ -91,6 +93,8 @@ ceph dashboard set-ganesha-clusters-rados-pool-namespace <cluster_id>:<pool_name
 * `pool`: The pool where ganesha recovery backend and supplemental configuration objects will be stored
 * `namespace`: The namespace in `pool` where ganesha recovery backend and supplemental configuration objects will be stored
 
+> **NOTE**: The RADOS settings aren't used in Ceph versions equal to or greater than Pacific 16.2.6, default values are used instead ".nfs" for the RADOS pool and the CephNFS CR's name for the RADOS namespace. However, RADOS settings are mandatory for versions preceding Pacific 16.2.6.
```

Comment:
This should be part of the release note too. Probably in the breaking change section.

Reply:
Is there something I can do about this on my end?

Reply:
If it were in a minor release (v1.8), we would add it to the PendingReleaseNotes.md, but this will need to be in a patch release so I'll make a mental note for a special release note. Do we have an idea how soon 16.2.6 will be out? It would be good to include this in the rook release immediately before that ceph release.

Reply:
Why not just waiting for v1.8?

Reply:
Rook will not be usable with NFS in v16.2.6 or newer until we merge this. Since v16.2.6 is about to be released, we should get this in the next v1.7.x patch.